packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -202,11 +202,22 @@
                     <oui-tile-description>
                         <oui-tile-description>
                             <span
+                                ng-if="$ctrl.getCurrentFlavor().minDiskSize === $ctrl.getCurrentFlavor().maxDiskSize"
                                 data-translate="pci_databases_general_information_node_type"
                                 data-translate-values="{
                                 coreNumber: $ctrl.getCurrentFlavor().core,
                                 memorySize: $ctrl.getCurrentFlavor().memory,
-                                storageSize: $ctrl.getCurrentFlavor().storage
+                                storageSize: $ctrl.getCurrentFlavor().minDiskSize
+                            }"
+                            ></span>
+                            <span
+                                ng-if="$ctrl.getCurrentFlavor().minDiskSize !== $ctrl.getCurrentFlavor().maxDiskSize"
+                                data-translate="pci_databases_general_information_node_type_disk_range"
+                                data-translate-values="{
+                                coreNumber: $ctrl.getCurrentFlavor().core,
+                                memorySize: $ctrl.getCurrentFlavor().memory,
+                                storageMinSize: $ctrl.getCurrentFlavor().minDiskSize,
+                                storageMaxSize: $ctrl.getCurrentFlavor().maxDiskSize,
```

Comment:
Related to this comment : 

```suggestion
                                storageMinSize: $ctrl.getCurrentFlavor().minDiskSize | cucBytes:2:false:'GB', // GB if API provide base unit value as GB, if default base is B, so use B and so on
                                storageMaxSize: $ctrl.getCurrentFlavor().maxDiskSize | cucBytes:2:false:'GB',
```

Reply:
Done here : https://github.com/ovh/manager/pull/5932/commits/ee7b1386639352dac98ee930dcb1e2b39891123c